apps/site/vercel-install.sh

```diff
@@ -3,7 +3,7 @@
 set -euxo pipefail
 
 # Change to the root directory
-pushd "$(dirname "$0")/.."
+pushd "$(dirname "$0")/../.."
```

Comment:
Wild idea, but would `git rev-parse --show-toplevel` work instead of us having to put the path? :smile:

Reply:
Great idea, I wish it was mine 😄 
[Use git rev-parse](https://github.com/blockprotocol/blockprotocol/pull/1289/commits/7d86e43d2823837bbe10105f23ceb708397b5e2a)